src/plugins/intel_cpu/thirdparty/llmdnn/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.18)
+
+project(root)
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+option(BUILD_TESTS "Build with tests" ON)
+
+message(INFO "--------------------------------")
+message(STATUS "Build with tests: ${BUILD_TESTS}")
+message(INFO "--------------------------------")
+
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
can we downgrade CXX standard ?
We support gcc-7 which does not support CXX17

Reply:
Done.